besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -3030,7 +3030,7 @@ protected void enableGoQuorumCompatibilityMode() {
 
   public static List<String> getJDKEnabledCypherSuites() {
     try {
-      SSLContext context = SSLContext.getInstance("TLS");
+      SSLContext context = SSLContext.getInstance("TLS 1.3");
```

Comment:
Looks like you'll need "TLSv1.3" here
https://docs.oracle.com/javase/7/docs/technotes/guides/security/StandardNames.html#SSLContext

Reply:
is there a test that we can add to ensure it's a valid value?